lib/beaker/host/windows/pkg.rb

```diff
@@ -65,7 +65,11 @@ def puppet_agent_dev_package_info( puppet_collection = nil, puppet_agent_version
     # - we do not have install_32 set on host
     # - we do not have install_32 set globally
     arch_suffix = should_install_64bit ? '64' : '86'
-    release_file = "puppet-agent-x#{arch_suffix}.msi"
+    if puppet_agent_version
+      release_file = "puppet-agent-#{puppet_agent_version}-x#{arch_suffix}.msi"
```

Comment:
I'm not sure if a similar change is needed for `pe_puppet_agent_promoted_package_info` below.